specificatie/kadastraal-onroerende-zaken.yaml

```diff
@@ -140,6 +140,8 @@ paths:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/403"
         '406':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/responses/406"
+        '410':
```

Comment:
een 410 kan alleen bij het opvragen van een enkele resource, niet bij een collectie. of is de hele collectie "gone"?

Reply:
Dat zat dan voor de wijziging uit #829 ook fout. Ik heb slechts hesrsteld wat ik in #829 had aangepast.